utils/build.ps1

```diff
@@ -1969,7 +1969,7 @@ function Build-BuildTools([Hashtable] $Platform) {
     -Platform $Platform `
     -UseMSVCCompilers $(if ($UseHostToolchain) { @("ASM_MASM", "C", "CXX") } else { @("") }) `
     -UsePinnedCompilers $(if ($UseHostToolchain) { @("") } else { @("ASM", "C", "CXX") }) `
-    -BuildTargets llvm-tblgen,clang-tblgen,clang-tidy-confusable-chars-gen,lldb-tblgen,llvm-config,swift-def-to-strings-converter,swift-serialize-diagnostics,swift-compatibility-symbols `
+    -BuildTargets llvm-tblgen,clang-tblgen,clang-tidy-confusable-chars-gen,lldb-tblgen,llvm-config,swift-def-to-strings-converter,swift-serialize-diagnostics,swift-compatibility-symbols,yaml2obj `
```

Comment:
This seems wrong. Why do we need to build yaml2obj for the build tools? The tests should be depending on `yaml2obj`. We just need to add it in the CMakeLists.txt at https://github.com/swiftlang/swift/blob/c75932b89fe79b6bedad79247f6ce8ae760dcb91/test/CMakeLists.txt#L112-L138.

Reply:
Good point.